go/mysql/server.go

```diff
@@ -38,6 +39,8 @@ const (
 	// timing metric keys
 	connectTimingKey = "Connect"
 	queryTimingKey   = "Query"
+
+	multiStatementError = "Do not allow MultiQueries '%v', Please set 'allowMultiQueries=true' if you are using jdbc connection."
```

Comment:
This is not only an issue for jdbc connections, so I would amend this statement to say something more generic like "Please set 'allowMultiQueries=true' to enable multiple statement support".